content/docs/install/multi-cluster/index.md

```diff
@@ -163,6 +163,13 @@ The important once that are used in the above example are:
 | `remoteControlPlane.api.token` | Keptn API token | `""` |
 
 
+## Known Limitations
+
+Keptn uses the integration name and version as identifier for connecting the integration to the control plane.
+Hence, in the case of multiple remote execution planes, each integration must have a different name/version pair, otherwise Keptn will assign the same set of subscriptions.
+To avoid this, we recommend to assign a different name to the integrations.
+Changing the integration name can be done by editing the Distributor `Deployment` manifest and change the environment variable `K8S_DEPLOYMENT_NAME`.
```

Comment:
I think we need more explanation about this "Deployment manifest".  K8S_DEPLOYMENT_NAME is listed on the https://keptn.sh/docs/0.18.x/reference/miscellaneous/distributor/ reference page --  is that what you want them to modify?
That page has a section of "Configuration examples" -- maybe we should add a little section about assigning a new name for a multi-cluster environment?